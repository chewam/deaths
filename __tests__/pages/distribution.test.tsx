/**
 * @jest-environment jsdom
 */
import messages from "@/lang/fr.json"
import { IntlProvider } from "react-intl"
import { render } from "@testing-library/react"

import Page from "../../src/pages/distribution"

test("Page snapshot: distribution", () => {
  jest.mock("next/router", () => ({
    useRouter() {
      return {
        locale: "fr",
        defaultLocale: "fr",
      }
    },
  }))

  const { asFragment } = render(
    <IntlProvider locale="fr" messages={messages}>
      <Page />
    </IntlProvider>
  )
  expect(asFragment()).toMatchSnapshot()
})
