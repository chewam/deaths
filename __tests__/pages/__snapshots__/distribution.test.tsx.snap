// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page snapshot: distribution 1`] = `
<DocumentFragment>
  <div
    class="container mx-auto px-6"
  >
    <div
      class="distribution"
    />
  </div>
</DocumentFragment>
`;
